dev-client/package.json

```diff
@@ -80,7 +80,7 @@
     "react-native-svg": "^15.7.1",
     "react-native-tab-view": "^3.5.2",
     "reduce-reducers": "^1.0.4",
-    "terraso-client-shared": "github:techmatters/terraso-client-shared#cbab96e",
+    "terraso-client-shared": "github:techmatters/terraso-client-shared#80ba146",
```

Comment:
Needs update before merge to main due to backend change log bug

Reply:
fyi: there won't be any changes to the schema associated with the bug fix so the client-shared commit on main right now should be stable for this PR